autoextract/aio/client.py

```diff
@@ -61,11 +160,16 @@ async def request_raw(query: Query,
 
     When handle_retries is True, this function can raise
 
-    1) autoextract.errors.ApiError, if there is an error returned by the API
-       which is not a throttling response (e.g. it can be raised for incorrect
-       request).
-    2) tenacity.RetryError, if a network-related error persists for
-       a long time, over the allowed time period.
+    1) autoextract.errors.RequestError,
+       if there is a Request-level error returned by the API
+       which is not a throttling response
+       (e.g. it can be raised for incorrect request).
+    2) autoextract.errors.QueryError,
```

Comment:
I don't think this method can raise QueryError. It shouldn't.

Reply:
That's true. QueryError exceptions should be catch and handled by the RequestProcessor. I've updated this docstring.